### PR TITLE
Chore/update out of date dependencies

### PR DESCRIPTION
**Pull request by madeline-scyphers**
# Summary

## Update
- Updated packages
  - Pillow
  - tensorflow
  - torch
  - torchvision
- Updated README instructions for how to run application

## Added
- Added geopandas to requirments.txt for non windows machines

## Removed
- Removed requirements.txts in specific pages subdirectories
because they should have their needed dependencies
installed from the base requirements.txt and it is a pain
to keep multiple copies up to date with each other.
- Removed h5py, and Keras explicit dependencies because it
seems the most recent version of tensorflow doesn't have
the same problem? Tested the image classifier page and
it classified fine on newer versions of tensorflow.
If we don't do that, then tensorflow won't work with
python 3.9, since all versions of tensorflow that work
with 3.9 require h5py > 3.0

### CODE REVIEW

**Comment by madeline-scyphers** [comment]
@MarcSkovMadsen I started with updating some of the dependencies to get it running with python 3.9 (libraries aren't out with 3.10 yet) and updated the read me just a little for some first steps on getting things at least runnable directly locally.

**Comment by MarcSkovMadsen** [comment]
I will take a look when my work day is over. Thanks so much.

**Comment by MarcSkovMadsen** [comment]
Feel free to connect on https://twitter.com/MarcSkovMadsen and https://www.linkedin.com/in/marcskovmadsen/ or gitter at @MarcSkovMadsen .

File: requirements_base.txt [hunk 1]
```diff
@@ -15,18 +15,16 @@ panel-highcharts==20210724.2
 
 # Data Engineering and Science
 pandas==1.2.2
-# geopandas # To `pip install geopandas` on windows please follow https://geoffboeing.com/2014/09/using-geopandas-windows/
+geopandas; platform_system != "Windows" # To `pip install geopandas` on windows please follow https://geoffboeing.com/2014/09/using-geopandas-windows/
```
Comment (MarcSkovMadsen) [inline requirements_base.txt:18]: I did not know this one. Thanks

**Comment by MarcSkovMadsen** [comment]
Basically everything looks fine.  But I need to get it tested. And I will use the opportunity to update to Panel 0.12.6 and get the README.md up to date. I have started.

For some reason the docker image that I build fails. So something must have changed since I build it last

```bash
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/tornado/web.py", line 1704, in _execute
    result = await result
  File "/usr/local/lib/python3.7/site-packages/panel/io/server.py", line 221, in get
    template_variables=session.document.template_variables
  File "/usr/local/lib/python3.7/site-packages/panel/io/server.py", line 151, in server_html_page_for_session
    bundle = bundle_resources(session.document.roots, resources)
  File "/usr/local/lib/python3.7/site-packages/panel/io/resources.py", line 140, in bundle_resources
    ext = bundle_models(None)
  File "/usr/local/lib/python3.7/site-packages/bokeh/util/compiler.py", line 308, in bundle_models
    _bundle_cache[key] = bundle = _bundle_models(custom_models)
  File "/usr/local/lib/python3.7/site-packages/bokeh/util/compiler.py", line 517, in _bundle_models
    custom_impls = _compile_models(custom_models)
  File "/usr/local/lib/python3.7/site-packages/bokeh/util/compiler.py", line 494, in _compile_models
    compiled = nodejs_compile(impl.code, lang=impl.lang, file=impl.file)
  File "/usr/local/lib/python3.7/site-packages/bokeh/util/compiler.py", line 113, in nodejs_compile
    output = _run_nodejs([compilejs_script], dict(code=code, lang=lang, file=file, bokehjs_dir=bokehjs_dir))
  File "/usr/local/lib/python3.7/site-packages/bokeh/util/compiler.py", line 445, in _run_nodejs
    return _run(_nodejs_path(), argv, input)
  File "/usr/local/lib/python3.7/site-packages/bokeh/util/compiler.py", line 421, in _nodejs_path
    _nodejs = _detect_nodejs()
  File "/usr/local/lib/python3.7/site-packages/bokeh/util/compiler.py", line 413, in _detect_nodejs
    '("conda install nodejs" or follow https://nodejs.org/en/download/)')
RuntimeError: node.js v14.0.0 or higher is needed to allow compilation of custom models ("conda install nodejs" or follow https://nodejs.org/en/download/)
```

**Comment by MarcSkovMadsen** [comment]
I am also considering whether I should change from pip to conda. I would have less problems working I believe if I did.

**Comment by madeline-scyphers** [comment]
I now remember that I had the problem when I first pulled it down, like a month and a half ago. But a forgot about it. I updated my version of node and all was good. 

conda might cause less issues. I assume the plan is to maintain compatibility with at least a few versions of python back. Base conda comes with 3.8, so you can’t use some of the newest python features, but that is fine is you want the repo to have compatibility with python at least 3.8+ if not older.

**Comment by MarcSkovMadsen** [comment]
Thanks @madeline-scyphers . I've merged the PR. Thanks so much.

- I've updated the README.
- I've upgraded to python 3.9.9.

I plan to do some more including creating a Vision/ Roadmap Feature Request linking to sub Feature Requests. Any input on "why" this project should exist and "what" it should be feel free to comment/ contribute. It will really help.

I don't get very much feedback on this project. So I don't know if it works at all for other users than me or not. Thanks.

**Pull request merged**